src/balena.sbvr

```diff
@@ -156,6 +156,9 @@ Term: last vpn event
 Term: local id
 	Concept Type: Short Text (Type)
 
+Term: logo
```

Comment:
```suggestion
Term: logotype
```

I prefer `logotype` but don't feel strongly about it as `logo` is also widely used.

Regardless, this seems to be just blob that we happen to store image data into, so either is not fitting. Maybe the term should be `logotype data`? I don't know, just thinking out loud. @Page- do you have an opinion?

Reply:
@dfunckt Since it's in `datauri` format, the idea is to have it interchangeable with a remote URL of the logo. What we decided is to store the logos as base64 encoded data in a datauri format for now as it is simpler to apply permissions against it (we don't want to expose private DT logos to everyone), and if this proves to not scale well, we can replace it with an S3 URL, for example.

Reply:
Maybe name it `logotype url` instead?

Reply:
AFAIK `logotype` refers only to the textual part of a logo, in this case we have both so it really is a logo. `logoUri` sounds fine though, I'll just wait for Page to chime in before making changes

Reply:
I'd never heard of `logotype` before @dfunckt but TIL `logo` is actually short for `logotype` (and also possibly for `logomark`), so my first reaction was "did you mean `logo type`?" as in the type of the logo. So fwiw I'd go with `logo`/`logo url` based on that

Reply:
I'm also confused by `logotype` but `logouri`/`logourl` sounds fine, just in case at some point we want to use `logo` for something else.

Reply:
I'll go with logo URI since URL is a bit inaccurate in this case